templates/registry/markdown/weaver.yaml

```diff
@@ -1,10 +1,20 @@
 templates:
-  - pattern: readme.md.j2
+  - pattern: attribute_readme.md.j2
     filter: '.groups | map(select(.type == "attribute_group")) | map(select(.id | startswith("registry"))) | group_by(.id | split(".") | .[1]) | map({id: .[0].id | split(".") | .[1], groups: .})'
     application_mode: single
+    file_name: 'registry/attributes/README.md'
```

Comment:
nit: there is some inconsistency around `registry/attribute` vs `registry/attributes` in the current commit

Reply:
Fixed